### DISCUSSION TITLE
"Use always a StringBuilder" - Internet myths

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#26] [General]
# "Use always a StringBuilder" - Internet myths

Use always a StringBuilder
That is what you can read from time to time. The basic idea is, that a StringBuilder is "better" ^to be defined^.
Why are people telling that ^lie^? Let's discuss this and see what it isn't true.

https://steven-giesel.com/blogPost/480539f1-98ab-45bc-ba24-9ccec65b459a

**Comment by borisStanojevic** [comment]
I like this one. There is a blog post by Jeff Atwood, which is similar in nature: https://blog.codinghorror.com/the-sad-tragedy-of-micro-optimization-theater/

**Reply by linkdotnet** [reply]
Wow. This article is almost 14 years old and it still holds true :)
